src/evo/assetlocktx.h

```diff
@@ -17,6 +17,7 @@
 class BlockManager;
 class CBlockIndex;
 class CRangesSet;
+class CTransaction;
```

Comment:
0c023f0a49d54b03bcb170a6b21bb126b85eb288: nit: `#include <primitives/transaction.h>` should have this covered already
```suggestion
```